### PR TITLE
Eyes Fixes and Chem Tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Just some fixes regarding eye blurriness and damage

- Internal Organ surgery adjusted for eyes
 - Internal organ surgery on the head will now remove nearsightedness and blindness, in addition to blurriness
- Internal organ surgery on the head will now also fix/repair deafness

- Oculine has been tweaked/adjusted a bit
 - 80% chance each cycle to reduce both eye AND ear damage by 1
 - 50% chance of curing nearsightedness
 - 30% chance of curing blindness
 - 30% chance to cure deafness if ear damage is less than 25

The above two should likely fix https://github.com/ParadiseSS13/Paradise/issues/2350

- Removes Audioline

- Doubles mitocholide's effectiveness
 - basically for each 2 units you consume, it'll heal 2 organ damage.

- Fixes being able to heal organs into negative damage thresholds.

- Fixes "3" being displayed whenever someone was dropped into various machines
 - Fixes https://github.com/ParadiseSS13/Paradise/issues/1601 this happens with visible message---more specifically, that's the message its supposed to display to blind individuals...thing there's no reason for the 3...it literally does nothing--anything after the comma is the message displayed to those who are blind....very strange.

### CODE REVIEW

**Comment by Alex6511** [comment]
Reviewing the code, Oculine probably should be updated to set the eye_blurry variable as well, it currently does not as far as I can tell.

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -354,7 +354,7 @@
 				usr << "[G:affecting.name] will not fit into the sleeper because they have a slime latched onto their head."
```
Comment (tigercat2000) [inline code/game/machinery/Sleeper.dm]: probably goes past the scope of this PR but this is a painful abuse of colons here

**Comment by Fox-McCloud** [comment]
@Alex6511

File: code/modules/admin/verbs/onlyoneteam.dm [hunk 2]
```diff
@@ -55,7 +55,7 @@
 
 		H.species.equip(H)
 		H.regenerate_icons()
-				
+
 	message_admins("[key_name_admin(usr)] used DODGEBAWWWWWWWL! -NO ATTACK LOGS WILL BE SENT TO ADMINS FROM THIS POINT FORTH-", 1)
```
Comment (tigercat2000) [inline code/modules/admin/verbs/onlyoneteam.dm]: should totes change this to 'dogebawwwwwwwl'

File: code/modules/surgery/organs_internal.dm [hunk 3]
```diff
@@ -156,6 +156,14 @@
 					"\blue You treat damage to [target]'s [I.name] with [tool_name]." )
 					I.damage = 0
 
+		if(target_zone == "head")
```
Comment (tigercat2000) [inline code/modules/surgery/organs_internal.dm]: snowflake

**Comment by Alex6511** [comment]
I forgot to actually check if you had included it.
What about the admin revive command? I don't see it there either.

**Comment by Crazylemon64** [comment]
I'd been wondering why I'd been seeing a 3 from time to time - thought it was some sort of ominous countdown, glad it was just a hiccup.

File: code/modules/surgery/organs_internal.dm (hunk 3 above)
Comment (TheDZD) [inline code/modules/surgery/organs_internal.dm]: Organs should probably just have an on_repair() proc or something that this surgery would then call.

Comment (Fox-McCloud) [inline code/modules/surgery/organs_internal.dm]: thing is a lot of these are completely decoupled from eyes--for instance, NEARSIGHTED and BLIND actually have...very little to do with eyes; blind and blurry, as well---and there's definitely not an "ears" organ. *shrug* Dunno.

**Comment by Spacemanspark** [comment]
My guess is that the three was just there for debugging purposes or something?

**Comment by tigercat2000** [comment]
Much more likely is that someone replaced show_message("shit", 1, "shitfuck", 2, "FUUUUUUU", 3) with visible_message()

**Comment by Fox-McCloud** [comment]
Made it more OOP, but I just had to go with the most ridiculous proc name ever.

**Comment by Crazylemon64** [comment]
I mean, the one thing this takes out is mediborgs mixing audioline in their patient's blood, though as hilarious as that was, I don't think it's that important.

**Comment by Fox-McCloud** [comment]
It made more sense to move ear curing into oculine than have it be its own chem---ear damage/deafness is a super niche damage type that very very rarely needs to be utilized---oculine is in a similar boat; it seemed like a waste of time to force chemists into making two super niche chems when just one niche chem (that handled both) would do fine.

That said, because it's more broad-spectrum, it's also not quite as strong as it used to be.
